csunplugged/config/urls.py

```diff
@@ -22,6 +22,7 @@
     url(r'', include('general.urls', namespace='general')),
     url(r'^topics/', include('topics.urls', namespace='topics')),
     url(r'^resources/', include('resources.urls', namespace='resources')),
+    url(r'^__dev__/', include('dev.urls', namespace='dev')),
```

Comment:
The URLs should only be available in `DEBUG` mode, not production. This should be moved to line 33.